vscode-trace-extension/src/trace-explorer/trace-tree.ts

```diff
@@ -113,31 +114,47 @@ const openDialog = async (): Promise<vscode.Uri | undefined> => {
 };
 
 export const fileHandler = (analysisTree: AnalysisProvider) => async (context: vscode.ExtensionContext, traceUri: vscode.Uri | undefined): Promise<void> => {
+    // Show the log output channel to the user while the trace is being processed
+    traceLogger.showLog();
+
+    // We need to resolve trace URI before starting the progress dialog because we rely on trace URI for dialog title
     if (!traceUri) {
         traceUri = await openDialog();
         if (!traceUri) {
-            console.log('Cannot open trace: invalid uri for trace', traceUri);
+            traceLogger.addLogMessage('Cannot open trace: invalid uri for trace ' + traceUri, fileHandler.name);
```

Comment:
This is not an error and we should not log it. traceUri is undefined when the user cancels the dialog.